samples/keys/identity_key_generation/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _identity_key_generate:
+
+Identity key generation
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to generate a random device-specific identity key and store in the Key
```

Comment:
```suggestion
This sample demonstrates how to generate a random device-specific identity key which is then stored in the Key
```

Reply:
Fixed!. Also full-line requirements for docs